mobile.css

```diff
@@ -0,0 +1,52 @@
+@media (max-width: 768px) {
+  .nav-list {
+    display: none;
+  }
+
+  .nav-bar {
+    padding: 10px;
+  }
+
+  .Introduction {
+    background: url(Assets/header.png) no-repeat center, #fff;
+    background-size: contain;
+    padding: 10px;
+  }
+
+  #hamburger {
+    align-self: flex-end;
+  }
```

Comment:
- [ ] Kindly make sure you start with the mobile approach as it's the standard. `@media screen and (min-width: 750px)`